src/Microsoft.AspNet.OData.Shared/Builder/StructuralTypeConfigurationOfTStructuralType.cs

```diff
@@ -97,6 +97,18 @@ public bool IsOpen
             }
         }
 
+        /// <summary>
+        /// Gets an indicator whether this EDM type is having instance annotations or not.
+        /// Returns <c>true</c> if this is having instance annotations; <c>false</c> otherwise.
+        /// </summary>
+        public bool IsWithInstanceAnnotations
```

Comment:
>public [](start = 8, length = 6)

Should this be public or internal? Do we expect customers to call this?

Reply:
#Resolved